packages/template-typescript-minimal/package.json

```diff
@@ -45,7 +49,185 @@
   },
   "mobify": {
     "app":{
-      "extensions": ["@salesforce/extension-sample"]
+      "extensions": [
+        "@salesforce/extension-sample",
+        "@salesforce/extension-store-locator",
```

Comment:
Here we can see that typescript-minimal is able to use this store-locator extension without passing in any config. And our extension-sdk will automatically merge in the default config for it. 

But in the near future, we plan to update our generator, so it will dump the default config in here for every extension used by the base project.